man/runc-update.8.md

```diff
@@ -0,0 +1,44 @@
+# NAME
+   runc update - update container resource constraints
+
+# SYNOPSIS
+   runc update [command options] <container-id>
+
+# DESCRIPTION
+   The data can be read from a file or the standard input, the
+accepted format is as follow (unchanged values can be omitted):
+
+   {
+     "memory": {
+       "limit": 0,
+       "reservation": 0,
+       "swap": 0,
+       "kernel": 0
+     },
+     "cpu": {
+       "shares": 0,
+       "quota": 0,
+       "period": 0,
+       "cpus": "",
+       "mems": ""
+     },
+     "blockIO": {
+       "blkioWeight": 0
+     },
+   }
+
+Note: if data is to be read from a file or the standard input, all
+other options are ignored.
+
+# OPTIONS
+   --resources, -r         path to the file containing the resources to update or '-' to read from the standard input.
+   --blkio-weight "0"      Specifies per cgroup weight, range is from 10 to 1000.
```

Comment:
It's copied from `runc update --help` and I think it's a cli problem, shall we address this in a separate PR?